modules/perception/camera/lib/traffic_light/detector/recognition/classify.cc

```diff
@@ -58,13 +60,12 @@ void ClassifyBySimple::Init(
 
   AINFO << "model_root" << model_root;
 
-  // TODO(zhangxiao): Create inference model
-  // rt_net_.reset(inference::CreateInferenceByName(
-  //     model_config.model_type(), proto_file, weight_file, net_outputs_,
-  //     net_inputs_, model_root));
-  // AINFO << "create success";
+  rt_net_.reset(inference::CreateInferenceByName(
+      model_config.model_type(), proto_file, weight_file, net_outputs_,
+      net_inputs_, model_root));
+  AINFO << "create success";
```

Comment:
And, pay attention to the use of Apollo logging.  Use `AINFO, ADEBUG, AWARNING, AERROR` etc to mark different levels of logging.